src/js/libgeo/GeoOps.js

```diff
@@ -1012,6 +1012,12 @@ geoOps.CircleBy3.updatePosition = function(el) {
 
 };
 
+geoOps.ArcBy3 = {};
+geoOps.ArcBy3.kind = "C";
+geoOps.ArcBy3.updatePosition = function(el) {
+    return geoOps.CircleBy3.updatePosition(el);
+};
+
```

Comment:
Just an idea: you could even shorten this to `geoOps.ArcBy3.updatePosition = geoOps.CircleBy3.updatePosition`. Would avoid one level of indirection, which shouldn't be noticeable in my opinion. So the main question is, which form is easier to understand at first glance? Yours fits in better with the pattern of the implementations around, while my suggestion makes it clear that the implementation is *exactly* the same. Pick that if you think the benefits outweight the drawbacks. Given that you came up with this here first, I doubt they do.